server/utils/regex.ts

```diff
@@ -0,0 +1 @@
+export const YOUTUBE_VIDEO_ID_REGEX = /(?:youtube\.com\/(?:[^\/]+\/.+\/|(?:v|e(?:mbed)?)\/|.*[?&]v=)|youtu\.be\/)([^"&?\/\s]{11})/;
```

Comment:
It may be worth commenting what this does. Or maybe I'm just bad at reading regexes

Reply:
True, will add a comment